urwid/wimp.py

```diff
@@ -32,21 +32,24 @@
 from urwid.split_repr import python3_repr
 from urwid.decoration import WidgetDecoration
 from urwid.command_map import ACTIVATE
+from urwid.constants import LEFT
 
 class SelectableIcon(Text):
     _selectable = True
-    def __init__(self, text, cursor_position=0):
+    def __init__(self, text, cursor_position=0, align=LEFT):
         """
         :param text: markup for this widget; see :class:`Text` for
                      description of text markup
         :param cursor_position: position the cursor will appear in the
                                 text when this widget is in focus
+        :param align: markup for this widget; see :class:`Text` for
+                      description of align
```

Comment:
Copy-paste whoopsie... align is not markup!

Instead of referring to `:class:Text` — just enumerate the possible values: `'left' (default), 'center', 'right'`, they're pretty self-explanatory